alphazero/_boardconverters.py

```diff
@@ -0,0 +1,19 @@
+import torch
+
+
+class BoardConverter:
+    @staticmethod
+    def board_to_tensor(board):
+        raise NotImplementedError
+
+
+class TicTacToeConverter(BoardConverter):
```

Comment:
should we actually put this stuff under `Game`?

Reply:
Don't know, my idea was that you could have multiple converters for different experiments for the same game, to see what works best. Might be not flexible to have it under Game then.

Reply:
ah, i see. that makes sense.